packages/Core/text.candy

```diff
@@ -9,21 +9,21 @@ is value := type.is value Text
 
 isEmpty text :=
   needs (is text)
-  ✨.textIsEmpty text
+  text | ✨.textIsEmpty
 length text :=
   # Returns the number of Unicode grapheme clusters in this text.
   needs (is text)
-  ✨.textLength text
+  text | ✨.textLength
 
 characters text :=
   needs (is text)
-  ✨.textCharacters text
+  text | ✨.textCharacters
 characterAt text index :=
   needs (is text)
   needs (int.is index)
   needs (int.isNonNegative index)
   needs (int.isLessThan index (length text))
-  struct.getUnwrap (characters text) index
+  text | characters | struct.getUnwrap index
```

Comment:
It actually did already since it used the `create_list(…)` function that changed from creating our struct-based list representation to a proper list, but this place wasn't updated. I agree that a list makes more sense for now, but we should probably switch to an iterable, at least when we have our own Unicode implementation in Candy